AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -48,12 +48,16 @@ public class CardComponent: PresentableComponent,
     public let supportedCardTypes: [CardType]
 
     /// Card component configuration.
-    public let configuration: Configuration
+    public var configuration: Configuration
```

Comment:
maybe make it `private(set)`? to prevent merchants from changing it from outside?